### PR TITLE
Plotting updates

### PR DESCRIPTION
**Pull request by jacobpennington**

### CODE REVIEW

File: nems/plots/timeseries.py [hunk 1]
```diff
@@ -66,3 +71,36 @@ def timeseries_from_epoch(signals, epoch, occurrences=0, channels=0,
...
     plot_timeseries(times, values, xlabel, ylabel, legend, ax=ax)
+
+
+def before_and_after(sig_name, rec, modelspec, idx, ax=None, title=None,
+                     channels=0, xlabel='Time', ylabel='Value'):
+    '''
+    Plots a timeseries of specified signal just before and just after
+    the transformation performed at some step in the modelspec.
+
+    Arguments:
```
Comment (bburan) [inline nems/plots/timeseries.py:82]: Should be `Parameters` (arguments are the values of function parameters).

File: nems/plots/timeseries.py [hunk 2]
```diff
@@ -66,3 +71,36 @@ def timeseries_from_epoch(signals, epoch, occurrences=0, channels=0,
...
+        The dataset to use. See nems/recording.py.
+
+    modelspec : list of dicts
+        The transformations to perform. See nems/modelspec.py.
+
+    idx : int
+        An index into the modelspec. rec[sig_name] will be plotted
+        as it exists after step idx-1 and after step idx.
+
+    Returns:
```
Comment (bburan) [inline nems/plots/timeseries.py:97]: I think you can leave this out if there's no return value (although, it's strictly true that all Python functions return `None` if there's no return statement). Either case, I don't feel strongly about this.

**Comment by bburan** [comment]
I haven't taken a deep look at the plotting code, so I can't provide too much feedback other than some comments on style. I didn't see any obvious issues though other than some docstring conventions (see my comments on the diff overview page).

**Comment by jacobpennington** [comment]
@bburan I don't see any comments on the "Files changed" page, and I'm not seeing something something called 'diff overview' - am I missing some github settings? Not sure how to see the comments you're talking a bout.

**Review by bburan — COMMENTED** [review]
Sorry! It's on the "files changed" tab. I think if you scroll down to the bottom of this page, you can see the comments I left as well. Two comments regarding the docstrings.

**Comment by jacobpennington** [comment]
Apparently I'm still bad managing branches correctly, just going to close this and pull a more recent one instead.

**Pull request closed without merging**